SoftLayer/CLI/routes.py

```diff
@@ -47,6 +47,7 @@
     ('config', 'SoftLayer.CLI.config'),
     ('config:setup', 'SoftLayer.CLI.config.setup:cli'),
     ('config:show', 'SoftLayer.CLI.config.show:cli'),
```

Comment:
I'd normally agree, but this is THE command that has to be run before using any other part of the CLI. It should be front and center. That's the motivation for putting it in the main CLI listing. The old location is also valid because that's properly namespaced. It involves the configuration of the CLI so it's under 'config'. There are only two subcommands there: show and config. I would be confused if I didn't see a way to edit the configuration there.